tests/common/integration_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 func init() {
 	testRunner = framework.IntegrationTestRunner
+	withAuth = WithAuth
```

Comment:
ditto

Reply:
integration's WithAuth to override default empty one.